### DISCUSSION TITLE
Matrix keypad 4x4

### DISCUSSION THREAD

**Discussion by jokokiswan** [openLuat/LuatOS#85] [Q&A] [answered]
Does luatos has already builtin for reading 4x4 matrix kepad? Thank you.

**Comment by chenxuuu** [comment, marked as answer]
only air105 support this


https://wiki.luatos.com/api/keyboard.html



---Original---
From: ***@***.***&gt;
Date: Thu, Jan 5, 2023 19:01 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [openLuat/LuatOS] Matrix keypad 4x4 (Discussion #85)




 
Does luatos has already builtin for reading 4x4 matrix kepad? Thank you.
